cloud/amazon/route53_health_check.py

```diff
@@ -0,0 +1,351 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: route53_health_check
+short_description: add or delete health-checks in Amazons Route53 DNS service
+description:
+     - Creates and deletes DNS Health checks in Amazons Route53 service
+     - Only the port, resource_path, string_match and request_interval can be updated
+version_added: "2.0"
+options:
+  state:
+    description:
+      - Specifies the action to take.
+    required: true
+    choices: [ 'present', 'absent' ]
+  ip_address:
```

Comment:
I can't actually get this module to work since I can't seem to get auth working with boto and anything over version 2.21.0, so I can't verify this, but doesn't that mean they both need to be optional, and then you just need a check that at least one of them is specified and fail if it isn't.  As it is now, since fqdn is required and you say it is forbidden if you use IP address, then wouldn't this simply not work with an IP address?